bio/dada2/filter-trim/meta.yaml

```
@@ -0,0 +1,15 @@
+name: dada2_filter_trim
+description: |
+  `DADA2 <https://benjjneb.github.io/dada2/index.html>`_
+  Quality filtering of single or paired-end reads using dada2 filterAndTrim function. Optional parameters are documented in the `manual <https://www.bioconductor.org/packages/release/bioc/manuals/dada2/man/dada2.pdf>`_.
+authors:
+  - Charlie Pauvert
+input:
+    fwd: a forward FASTQ file (potentially compressed) without primers sequences
+    rev: a (facultative) reverse FASTQ file (potentially compressed) without primers sequences
```

Minor preference of word choice in English (we also have `fakultativ` in German, so I see where this word choice is coming from;):
```suggestion
    rev: an (optional) reverse FASTQ file (potentially compressed) without primers sequences
```

---

Haha I'm busted!